django/contrib/redirects/migrations/0003_migrate_sites.py

```diff
@@ -0,0 +1,31 @@
+from django.db import migrations
+
+
+def migrate_site_domains(apps, schema_editor):
```

Comment:
I would suggest to merge this and the `0002` migration into one.

Reply:
Schema change and data migration should be separated, see @knbk comment above.

Reply:
ok, fair enough